Files/Filesystem/LibraryManager.cs

```diff
@@ -110,7 +110,6 @@ private async Task SyncLibrarySideBarItemsUI()
                             MainPage.SideBarItems.Insert(1, librarySection);
 
                             libraryItems.Clear();
-                            libraryItems.Add(AppSettings.DesktopPath);
```

Comment:
Why? It was useful to me!

Reply:
@d2dyno1 The desktop path isn't a default library, until we add proper support for libraries, I think we should try to follow the default behavior.

Reply:
Then should we move desktop folder to the first section with the recyclebin?

Reply:
@gave92 The desktop location is already one of the default locations pinned to the favorite section.

Reply:
Keep forgetting about that, for me it's a yes